MinVer.Lib/VersionSettings.cs

```diff
@@ -0,0 +1,14 @@
+namespace MinVer.Lib
+{
+    using System.Collections.Generic;
+
+    public sealed class VersionSettings
```

Comment:
That's a fair point, I originally went down that route, but when passing it (`defaultPreReleaseIdentifers`) into the `Version` constructor it felt weird because that sort of like a value type, and here I am passing in something that isn't really like the other parameters, it's only used under certain scenarios.

Reply:
I think that's OK. With the changes I pushed to https://github.com/adamralph/minver/compare/default-pre-release-version-changes?expand=1, when calling the constructors, the argument is not just the _default_ pre-release, it _is_ the pre-release (btw - I just pushed another commit to that branch to reflect that) and when calling `WithHeight()` the parameter name `defaultPreReleaseIdentifers` expresses the fact that these are the _default_ pre-release identifiers, i.e. they should be _if_ pre-release identifiers are required during the execution of that method.

Reply:
^ irrelevant now that the constructor changes are not required

Reply:
https://github.com/adamralph/minver/compare/default-pre-release-version-changes?expand=1 still contains constructor changes, but _different_ ones. 🙂